website_event_compassion/forms/donation_form.py

```diff
@@ -128,6 +128,9 @@ def form_before_create_or_update(self, values, extra_values):
             })
 
         def _form_create(self, values):
+            # Set payment term as immediate
+            values['payment_term_id'] = self.env.ref(
```

Comment:
Why not including at line 127 where we already define all invoice values?

Reply:
Could you check it?

Reply:
Yes it's done